Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -321,11 +321,13 @@ namespace xr
     {
         const System::Impl& SystemImpl;
         std::vector<Frame::View> ActiveFrameViews{ {} };
-        std::vector<Frame::InputSource> InputSources{};
+        std::vector<Frame::InputSource> InputSources;
```

Comment:
I'm guessing this change was unintentional?

Reply:
Yes unintentional thanks fixing.